newrelic.js

```diff
@@ -9,22 +9,25 @@ exports.config = {
   /**
    * Array of application names.
    */
-  app_name: ['mjml-server'],
+  app_name: ["mjml-server"],
   /**
    * Your New Relic license key.
    */
-  license_key: 'license key here',
+  license_key: "license key here",
```

Comment:
Is this working?

Reply:
yeah, NewRelic loads things from the ENV with priority over this file (this is set on a k8s secret env)